superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -638,7 +638,9 @@ class SqlEditor extends React.PureComponent {
                         this.props.defaultQueryLimit,
                     )}
                   </span>
-                  <Icon name="triangle-down" />
+                  <Icons.TriangleDown
+                    iconColor={supersetTheme.colors.grayscale.base}
```

Comment:
Hi @pkdotson the icons should inherit the color of the container? Is it here a particular issue for which you need to specify the color while with the previous implementation it wasn't necessary?

Reply:
actually in this case they do not inherit the colors from their container. I had to specifically use the prop because the default color was almost black.

Reply:
@geido / @pkdotson does this indicate an issue with the `Icons` component that needs to be addressed? Not sure what makes this particular use case special if it's not inheriting the color as intended.

Reply:
So I checked this @pkdotson @rusackas. In reality, the icon is correctly inheriting the color of the container, which is black. But the icon here is intended to be #666 reason why Phillip specified. With the previous icon implementation, that wasn't necessary for the simple fact that #666 was the default color of the icons. We might make it the default also for the new icon implementation if that can avoid specifying it all the time.

Reply:
@geido I agree. The default color should be one defined in the theme. A variation of `grayscale` or `primary`.

Reply:
Will leave the icon color for now as we come up with a solution for default color for icons